rpcs3/Emu/RSX/Overlays/overlay_user_list_dialog.cpp

```diff
@@ -243,33 +244,15 @@ namespace rsx
 			this->on_close = std::move(on_close);
 			visible = true;
 
-			auto& list_thread = g_fxo->get<named_thread<user_list_dialog_thread>>();
```

Comment:
user_list_dialog_thread can now be removed

Reply:
Done.